courseraprogramming/commands/config.py

```diff
@@ -20,15 +20,77 @@
 You may install it from source, or via pip.
 """
 
+from courseraprogramming.commands import oauth2
+import requests
 import logging
+import time
 import sys
 
 
-def command_config(args):
-    "Implements the configure subcommand"
-    # TODO: IMPLEMENT ME!
-    logging.fatal('The upload subcommand is not implemented! :-( Sorry!')
-    sys.exit(3)
+def check_auth(args):
+    """
+    Checks courseraprogramming's connectivity to the coursera.org API servers
+    """
+    oauth2_instance = oauth2.build_oauth2(args)
+    auth = oauth2_instance.build_authorizer()
+    my_profile_url = (
+        'https://api.coursera.org/api/externalBasicProfiles.v1?'
+        'q=me&fields=name'
+    )
+    r = requests.get(my_profile_url, auth=auth)
+    if r.status_code != 200:
+        logging.error('Received response code %s from the basic profile API.',
+                      r.status_code)
+        logging.debug('Response body:\n%s', r.text)
+        sys.exit(1)
+    try:
+        external_id = r.json()['elements'][0]['id']
+    except:
+        logging.error(
+            'Could not parse the external id out of the response body %s',
+            r.text)
+        external_id = None
+
+    try:
+        name = r.json()['elements'][0]['name']
+    except:
+        logging.error(
+            'Could not parse the name out of the response body %s',
+            r.text)
+        name = None
+
+    if not args.quiet > 0:
+        print 'Name: %s' % name
+        print 'External ID: %s' % external_id
+
+    if name is None or external_id is None:
+        sys.exit(1)
+
+
+def display_auth_cache(args):
+    '''
```

Comment:
you're inconsistent in your docstring format. pick either single or double quotes. (pep8 says double quotes : http://legacy.python.org/dev/peps/pep-0008/#documentation-strings)